### PR TITLE
[v1.9.0] Document the collections SFTP requirement and file tagging. 

### PR DESCRIPTION
**Pull request by ddimatos**
##### SUMMARY
Addresses issue #1280 which explains that several modules require SFTP and the expected behavior.
Also updates zos_archive doc which has been updated to reference `src` over `path`.
Also includes doc update to included the added plugins.
Addresses issue #1178 which explains how existing files will have the file tag updated to the managed nodes locale or the value set in the encoding option. 

##### ISSUE TYPE
- Docs Pull Request

##### COMPONENT NAME
 - [x] `zos_copy`
  - [x] `zos_fetch`
  - [x] `zos_script`
  - [x] `zos_job_submit`
  - [x] `zos_unarchive`

### CODE REVIEW

**Comment by ddimatos** [comment]
zos_script:
image

**Comment by ddimatos** [comment]
zos_fetch:
image

**Comment by ddimatos** [comment]
zos_job_submit:
image

**Comment by ddimatos** [comment]
zos_copy
image

**Comment by ddimatos** [comment]
zos_uncarchive
image

File: docs/source/plugins.rst [hunk 1]
```diff
@@ -5,30 +5,33 @@
 Plugins
 =======
 
-Plugins that come with the **IBM z/OS core collection** augment Ansible's core
+Plugins that come with the **IBM z/OS core collection** compliment Ansible's core
```
Comment (richp405) [inline docs/source/plugins.rst:8]: compliment with i = "says something nice about"
complement with e = "improves upon something else"
I think you need the 'e' here... it just jumped out at me.

**Review by richp405 — APPROVED** [review]
Minor/grammatical issue in plugins.rst.  Code looks fine

**Pull request merged**